docs/key_features.md

```diff
@@ -5,16 +5,19 @@
 - Works with labelled, n-dimensional data (e.g., geospatial, vertical and temporal dimensions) for both point-based and gridded data. `scores` can effectively handle the dimensionality, data size and data structures commonly used for:
   - gridded Earth system data (e.g., numerical weather prediction models)
   - tabular, point, latitude/longitude or site-based data (e.g., forecasts for specific locations).
-- Handles missing data, masking of data and weighting of results.
-- Supports [xarray](https://xarray.dev/) datatypes, and works with [NetCDF4](https://doi.org/10.5065/D6H70CW6), [HDF5](https://github.com/HDFGroup/hdf5), [Zarr](https://zarr.dev) and [GRIB](https://codes.wmo.int/grib2) data sources among others. 
+- Handles missing data, masking of data and [weighting of results](project:./tutorials/Weighting_Results.md) (e.g. by area, by latitude, by population).
+- Supports [xarray](https://xarray.dev/) datatypes, and works with [NetCDF4](https://doi.org/10.5065/D6H70CW6), [HDF5](https://github.com/HDFGroup/hdf5), [Zarr](https://zarr.dev) and [GRIB](https://codes.wmo.int/grib2) data formats among others. 
 
 ## Usability
 
 - A companion Jupyter Notebook [tutorial](project:./tutorials/Tutorial_Gallery.md) for each metric and statistical test that demonstrates its use in practice.
 - [Over 60 metrics, statistical techniques and data processing tools](included.md), including:
   - commonly-used metrics (e.g., mean absolute error (MAE), root mean squared error (RMSE))
   - novel scores not commonly found elsewhere (e.g., FIxed Risk Multicategorical (FIRM) score ([Taggart et al., 2022](https://doi.org/10.1002/qj.4266)), Flip-Flop Index ([Griffiths et al., 2019](https://doi.org/10.1002/met.1732), [2021](https://doi.org/10.1071/ES21010)))
-  - complex scores (e.g., threshold-weighted continuous ranked probability score (twCRPS))
+  - recently developed, user-focused scores and diagnostics including:
+      - threshold-weighted scores, such as threshold-weighted continuous ranked probability score (twCRPS) [(Gneiting and Ranjan 2011)](https://doi.org/10.1198/jbes.2010.08110) and threshold-weighted mean squared error (MSE) [(Taggart 2022)](https://doi.org/10.1002/qj.4206)
+      - Murphy diagrams [(Ehm et al., 2016)](https://doi.org/10.1111/rssb.12154)
+      - isotonic regression for reliability diagrams [(Dimitriadis et al., 2021)](https://doi.org/10.1073/pnas.2016191118)
```

Comment:
@nicholasloveday - I updated this section as per your helpful suggestion back on 3/8/24. As some time has since passed, and more metrics have since been added/will shortly be added, I wasn't sure if you wished to amend this, or leave it as is. Please let me know if you would like any changes to be made.